.github/workflows/deploy.yml

```
@@ -152,12 +152,37 @@ jobs:
         
         # Setup environment
         echo "⚙️  Configuring environment..."
-        if [ ! -f .env ]; then
-          echo "Creating .env file..."
-          echo "PORT=3002" > .env
-          echo "NODE_ENV=production" >> .env
-          echo "DATABASE_URL=${{ secrets.DATABASE_URL }}" >> .env
-        fi
+        cat > .env << EOF
+# Onasis Gateway Production Environment
+NODE_ENV=production
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 157: could not find expected ':'

(syntax-check)